crypto/evp/bio_enc.c

```diff
@@ -61,6 +61,9 @@ const BIO_METHOD *BIO_f_cipher(void)
     return &methods_enc;
 }
 
+extern BIO_ENC_CTX *ctxcpy;
```

Comment:
I wonder why this is here?  It doesn't seem to be read anywhere and is only written to below.
A global like this seems like poor practice...

Reply:
Oops. A left over debugging artefact. Removed now.